lib/mayaUsd/render/px_vp20/CMakeLists.txt

```diff
@@ -23,5 +23,5 @@ mayaUsd_promoteHeaderList(HEADERS ${headers} SUBDIR render/px_vp20)
 # install
 # -----------------------------------------------------------------------------
 install(FILES ${headers}
-    DESTINATION ${CMAKE_INSTALL_PREFIX}/include/mayaUsd/render/px_vp20
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/include/${PROJECT_NAME}/render/px_vp20
```

Comment:
We have a lot of these, which are kind of hard-coded paths. I wonder if there is a cmake variable or way to use the relative path from project_name? Such that we could replace "${PROJECT_NAME}/render/px_vp20" with a variable.